src/api/status/src/js/queue-stats.js

```diff
@@ -0,0 +1,13 @@
+const { fetch } = require('@senecacdot/satellite');
+
+module.exports = async function getJobCount() {
+  let jobCount = 0;
+  try {
+    const data = await fetch(`${process.env.API_URL}v1/posts/feeds/info`, { method: 'GET' });
```

Comment:
The fetching might not work on staging/production. But this can be merged, I will be fixing this issue since the total post, and total feed also not working.

Reply:
Yeah I just copied the hbs partials. At least they are similar

Reply:
@TueNguyen2911 Actually maybe once my PR #2590 is merged, you can rebase and fix the link

Reply:
It should be a simple
```suggestion
    const data = await fetch(`${process.env.POSTS_URL}/feeds/info`, { method: 'GET' });
```